SwiftUIDZ/SwiftUIDZ/FiltersView.swift

```diff
@@ -0,0 +1,163 @@
+// FiltersView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран фильтров
+struct FiltersView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let navBarHeight: CGFloat = 10
+        static let backButtonIconName = "chevron.left"
+        static let navigationTitle = "Filters"
+        static let filtersImageName = "filter"
+        static let categoryText = "Category"
+        static let moreText = "More"
+        static let arrowImageName = "chevron.right"
+        static let orangeColorName = "pastelOrange"
+        static let pricingText = "Pricing"
+        static let colorsText = "Colors"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        VStack {
+            GradientNavigationView(navBarHeight: Constants.navBarHeight)
+            categoryTextView
+            productLazyHGrid
+            makeTextView(text: Constants.pricingText)
+            sliderView
+            colorsTextView
+            colorsLazyVGrid
+            Spacer()
+        }
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .navigation) {
+                HStack(spacing: 65) {
+                    Button {
+                        presentationMode.wrappedValue.dismiss()
+                    } label: {
+                        Image(systemName: Constants.backButtonIconName)
+                    }
+                    Spacer()
+                    Text(Constants.navigationTitle)
+                        .foregroundColor(.white)
+                        .font(.title3)
+                        .fontWeight(.bold)
+                    Spacer()
+                    Image(Constants.filtersImageName)
+                        .resizable()
+                        .renderingMode(.template)
+                        .foregroundColor(.white)
+                        .frame(width: 20, height: 20)
+                }
+            }
+        }
+        .accentColor(.white)
+    }
+
+    // MARK: - Private Properties
+
+    @Environment(\.presentationMode) var presentationMode
+
+    @StateObject private var viewModel = FiltersViewModel()
+
+    private var oneColumnGrid = [GridItem(.fixed(80))]
+    private var fiveColumnGrid = [
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70))
+    ]
+
+    private var colorsLazyVGrid: some View {
+        ScrollView(.vertical, showsIndicators: false) {
+            LazyVGrid(columns: fiveColumnGrid, spacing: 20) {
+                ForEach(Colors.colors, id: \.self) { color in
+                    Button {
+                        viewModel.color = color
+                    } label: {
+                        Circle()
+                            .fill(color)
+                            .frame(width: 40, height: 40)
+                    }
```

Comment:
```suggestion
                    Button {
                        viewModel.color = color
                    } label: {
                        Circle()
                            .fill(color)
                            .frame(width: 40, height: 40)
                    }
```
private var ...
я стараюсь каждый такой эл-т отдельно писать